gwdetchar/lasso/core.py

```diff
@@ -42,18 +42,32 @@ def find_outliers(ts, N=5):
         data to find outliers within
 
     N : `float`, optional
-        number of standard deviations to consider an outlier, default: 5
+        if `method='s'`: number of standard deviations to consider an outlier
+        if `method='pf'`: percentile range limit to consider an outlier
+        default value for both methods:
```

Comment:
This seems to be missing a default value:

```suggestion
        default for both methods: 5
```